### ISSUE TITLE
Sort, Filter and pagination icons in datagrid should have title set

### ISSUE THREAD

**Issue by immihaylova** [vmware/clarity#4454]
## Describe the bug

This is an issue with accessibility. We got a report from AccessLevel where they note that icons inside a button or a link should be marked as decorative.

The clr-icon-s for sort and filter in the datagrid column are inside a button but are not properly marked as decorative, and because of this the accessibility reader tries to read the svg inside them.

Currently, the icons are marked with role=none but this only hides their semantics, and not their structure. This means that all their child nodes are being read by the reader. 
Since, the svg inside the clr-icon has a role=img, the reader expects it to have a label it can read.

In order to make the reader ignore the child nodes of the clr-icon, attr.aria-hidden=true should be added to it. This will make the reader ignore all its child nodes.

I have attached a screenshot with the items properties shown in debugger.

## How to reproduce

Simply use clr-dg-column component with sort and filter.

1.  Go to a view containing clr-dg-column
2.  Right-Click on sort of filter icon and choose Inspect.
3.  Check its accessibility properties in the debugger.

## Expected behavior

<clr-icon role=none attr.aria-hidden=true...> 

## Versions

**App**

* Angular: [9]
* Node: [10]
* Clarity: [3]

**Device:**

* Type: [PC]
* OS: [Win10]
* Browser [chrome]
* Version [80]
![clarity_sort_icon](https://user-images.githubusercontent.com/43816378/78766947-a04fec80-7992-11ea-8c83-cfeee9a00dc3.png)
![clarity_filter_icon](https://user-images.githubusercontent.com/43816378/78767008-b65dad00-7992-11ea-9542-af17af7e2f66.png)

**Comment by immihaylova** [comment]
I received an update from AccessLevel stating just the opposite for these icons. Тhey want the icons to have a title and not be decorative. The Show/Hide Column buttons are done this way, and according to AccessLevel, this is the right way to do it. Here is their input:

> Hi Ivelina,

That is a slight oversight on my part, my apologies. Thank you for pointing it out. I will correct that to make it consistent with the rest of the report. **The sort arrows should not be decorative as it can be used as a guide for sighted users including keyboard only users if the sort is ascending or descending.**

and

> Adding title attributes to the pagination icons would be highly recommended as there are no adjacent text to visually and explicitly explain its purpose.

**Comment by immihaylova** [comment]
Here are screenshots of the pagination buttons that need to be fixed as well.
![pg_vms_list_pagination](https://user-images.githubusercontent.com/43816378/80091864-8275b080-856a-11ea-8357-19b68428403e.png)


And here is a screenshot of Show/Hide Column button as an example of the desired accessibility properties:
![showHideColumns](https://user-images.githubusercontent.com/43816378/80091920-99b49e00-856a-11ea-9ff4-6c9296e787dd.png)

**Comment by coryrylan** [comment]
Aria label and title attr should not be on an icon directly when inside an icon only button. The fix for this is to put the title on the surrounding button to match with the `aria-label` as that is the action element we are describing. We can update the example here https://clarity.design/icons/accessibility

```html
<button class="btn btn-icon" aria-label="main menu" title="main menu">
  <clr-icon shape="bars"></clr-icon>
</button>
```

Related issue https://github.com/vmware/clarity/issues/4508

**Comment by immihaylova** [comment]
Thanks, Cory.

In this case, Show/Hide Columns should also be fixed to match the described state.

Also, it seems like 'aria-hidden=true' needs to be added to the clr-icon element, so the svg inside it won't be visible to screen readers.

<button class="btn btn-icon" aria-label="main menu" title="main menu">
   <clr-icon shape="bars" aria-hidden=true></clr-icon>
</button>
